bundles/nl.esi.comma.project.standard.cli/dist/start-simulator.bat

```diff
@@ -1,36 +1,14 @@
 @ECHO OFF
 SETLOCAL
 
-IF NOT EXIST "%1" (
-  echo File does not exists: %1
-  exit 1
+FOR /F "delims=" %%i IN ("%~dp0") DO (
+  set script_drive=%%~di
+  set script_path=%%~pi
 )
 
-FOR /F "delims=" %%i IN ("%1") DO (
-  set file_drive=%%~di
-  set file_path=%%~pi
-  set file_name=%%~ni
-  set file_extension=%%~xi
-)
-
-set project_file=%file_drive%%file_path%%file_name%.prj
-echo Creating project:  %project_file%
-
-(
-  echo Project project {
-  echo   Generate Simulator {
-  echo     simulator {
-  echo       product-file "%file_name%%file_extension%"
-  echo     }
-  echo   }
-  echo }
-) > "%project_file%"
+cd models
 
-echo Generating simulator
-.\jre\bin\java.exe -jar .\bpmn4s-generator.jar -l "%project_file%"
-
-set simulator_file=%file_drive%%file_path%src-gen\CPNServer.py
+set simulator_file=%script_drive%%script_path%\models\CPNServer.py
```

Comment:
CPNServer.py is not located in the models dir but in the dist (i.e. root) dir, so why is this magic paths needed?